scripts/setup.sh

```diff
@@ -20,13 +28,23 @@ if ! setup_service yarn 1.22.19 &> /dev/null; then
   exit ${FAILED_SETUP}
 fi
 
-cd ${OKTA_HOME}/${REPO}
+cd "${OKTA_HOME}"/"${REPO}" || exit
```

Comment:
Just curious, why  this change is needed?

Reply:
Ah that was leftover from copying some logic from a different script, it's not necessary I can change it back to minimize changes